geospaas_processing/copiers.py

```diff
@@ -2,6 +2,7 @@
 import shutil
 import logging
 import time
+from distutils.dir_util import copy_tree
```

Comment:
There is already a `copytree()` function in `shutil`, no need for an extra import.

Reply:
Done

Reply:
Done